src/pytest_html/nextgen.py

```diff
@@ -0,0 +1,66 @@
+import json
+from typing import Any
+from typing import Dict
+
+import pytest
+
+
+class NextGenReport:
+    def __init__(self, config, data_file):
+        self._config = config
+        self._data_file = data_file
+
+        self._title = "Next Gen Report"
+        self._data = {
+            "title": self._title,
+            "collectedItems": 0,
+            "environment": {},
+            "tests": [],
+        }
+
+        self._data_file.parent.mkdir(parents=True, exist_ok=True)
+
+    def _write(self):
+        try:
+            data = json.dumps(self._data)
+        except TypeError:
+            data = cleanup_unserializable(self._data)
+            data = json.dumps(data)
+
+        with self._data_file.open("w", buffering=1, encoding="UTF-8") as f:
+            f.write("const jsonData = ")
```

Comment:
can we combine these `f.write` calls on an f string line `f"const jsonData = {data}\n"`?